src/api/spec/models/token/workflow_spec.rb

```diff
@@ -1,6 +1,42 @@
 require 'rails_helper'
 
 RSpec.describe Token::Workflow do
+  describe '#workflow_token_users' do
+    let(:token_user) { create(:confirmed_user) }
+
+    context 'asking for workflow tokens' do
+      let(:workflow_token) { create(:workflow_token, user: token_user) }
+
+      before { token_user.shared_workflow_tokens << workflow_token }
+
+      it 'returns the users whom I shared the token' do
+        expect(workflow_token.users_shared_among).to include(token_user)
```

Comment:
What are you testing here? AR associations? I'm pretty sure they do that in the AR spec :) CI time is developer time.